server/Gemfile

```diff
@@ -25,10 +25,10 @@ gem "puma", "~> 5.0"
 # gem "bcrypt", "~> 3.1.7"
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
-gem "tzinfo-data", platforms: %i[ mingw mswin x64_mingw jruby ]
+gem 'tzinfo-data', platforms: %i[ mingw mswin x64_mingw jruby ]
```

Comment:
Just curious why are there formatting changes.

Reply:
Ah I have a linter installed locally and it was picking up these style things. In Ruby we use `"#{variable}"`, or ``${variable}`` in JS.
`'#{variable}'` will not work for Ruby!

Reply:
I am a bit confused so it's changing to the `'#{variable}'` which might not work in Ruby?

Reply:
My bad lol, that was a very confusing explanation. Basically the linter is just telling me "don't use `""` when you don't need to because `""` is generally used for template literals"

Reply:
I see. Thank you!